bin/install-core

```diff
@@ -160,17 +189,17 @@ if (( old_versions != versions )) || [ ! -f $LOCAL_DIR/bin/bitcoind ]; then
         if [ ! -f SHA256SUMS ]; then
             zenity --warning --title="Signature file not found" --text="Click OK to try downloading Bitcoin Core again." --ellipsize $ICON
```

Comment:
@epiccurious: same for the signature file

Reply:
Resolved with https://github.com/BenWestgate/Bails/pull/94/commits/c2a241db58bbcd5d032271e36db659aba4df7307.